README.md

```diff
@@ -16,26 +16,13 @@ dotnet add package HybridModelBinding
 
 #### Startup.cs
 
-```csharp
```

Comment:
Please retain the opening code-format block.

Reply:
oops. my mistake